plot_data/core.py

```diff
@@ -1104,16 +1116,16 @@ class Arc2D(PlotDataObject):
     :type edge_style: EdgeStyle
     """
 
-    def __init__(self, cx: float, cy: float, r: float, start_angle: float, end_angle: float, clockwise: bool = None,
-                 edge_style: EdgeStyle = None, name: str = ''):
+    def __init__(self, cx: float, cy: float, r: float, start_angle: float, end_angle: float, data=None,
```

Comment:
data argument has been added but seems unused. Is this wanted ?

Reply:
Thanks